### ISSUE TITLE
pbot sync throws method missing exception for octokit

### ISSUE THREAD

**Issue by andreasohlund** [Particular/PBot#75]
Are we out of sync?

### Command

sync NServiceBus.Callbacks target branch develop

### Exception

```
System.AggregateException: One or more errors occurred. ---> System.MissingMethodException: Method not found: 'Void Octokit.Internal.HttpClientAdapter..ctor(System.Net.IWebProxy)'.
  at SyncOMatic.GitHubGateway.ClientFrom(Credentials credentials, IWebProxy proxy)
  at SyncOMatic.GitHubGateway.SetupClientCache(IEnumerable`1 credentialsPerRepos, Credentials defaultCredentials, IWebProxy proxy)
  at SyncOMatic.GitHubGateway..ctor(Credentials defaultCredentials, IWebProxy proxy, Action`1 logCallBack)
  at SyncOMatic.Syncer..ctor(Credentials defaultCredentials, IWebProxy proxy, Action`1 loggerCallback)
  at PBot.SyncOMatic.SynchronizeRepo.<Execute>d__1.MoveNext()
  --- End of inner exception stack trace ---
  at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
  at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
  at PBot.BotCommand.<Register>b__3_0(IResponse`1 msg)
---> (Inner Exception #0) System.MissingMethodException: Method not found: 'Void Octokit.Internal.HttpClientAdapter..ctor(System.Net.IWebProxy)'.
  at SyncOMatic.GitHubGateway.ClientFrom(Credentials credentials, IWebProxy proxy)
  at SyncOMatic.GitHubGateway.SetupClientCache(IEnumerable`1 credentialsPerRepos, Credentials defaultCredentials, IWebProxy proxy)
  at SyncOMatic.GitHubGateway..ctor(Credentials defaultCredentials, IWebProxy proxy, Action`1 logCallBack)
  at SyncOMatic.Syncer..ctor(Credentials defaultCredentials, IWebProxy proxy, Action`1 loggerCallback)
  at PBot.SyncOMatic.SynchronizeRepo.<Execute>d__1.MoveNext()<---

```

**Comment by andreasohlund** [comment]
Packages.config says : `<package id="Octokit" version="0.18.0" targetFramework="net45" />`
and pbot says:

```

Andreas Öhlund [2:09 PM] 
list installed packages

Particular BotBOT [2:09 PM] 
agsXMPP - 1.1.1.0

[2:09] 
Bert.RateLimiters - 1.0.0

[2:09] 
Common.Logging - 2.1.2

[2:09] 
Common.Logging.Log4Net - 2.0.1

[2:09] 
Humanizer - 1.37.7

[2:09] 
log4net - 1.2.10

[2:09] 
Microsoft.Bcl - 1.1.9

new messages
[2:09] 
Microsoft.Bcl.Build - 1.0.14

[2:09] 
Microsoft.Net.Http - 2.2.28

[2:09] 
Microsoft.Owin - 2.0.2

[2:09] 
Microsoft.Owin.Host.HttpListener - 2.0.2

[2:09] 
Microsoft.Owin.Hosting - 2.0.2

[2:09] 
MMBot.Router.Nancy - 0.9.8.8

[2:09] 
MMBot.Slack - 0.9.8.7

[2:09] 
Nancy - 0.23.2

[2:09] 
Nancy.Owin - 0.23.2

[2:09] 
Newtonsoft.Json - 6.0.3

[2:09] 
Octokit - 0.18.0
```

so we seems to be ok there. Other ideas? @distantcam @adamralph

**Comment by distantcam** [comment]
Ah, I had to upgrade octokit to add a command. I guess we should also update syncomatic to use the same version of octokit.

**Comment by andreasohlund** [comment]
Ah, makes sense, will you handle it? (think we deploy syncomatic manually to myget)

**Comment by distantcam** [comment]
Yep I'll sort it out.

**Comment by distantcam** [comment]
This should be fixed now as part of pbot 0.1.160 and syncomatic 0.1.6

**Comment by andreasohlund** [comment]
It works ok now
